reference/opt/shaders-msl/vert/set_builtin_in_func.vert

```diff
@@ -0,0 +1,17 @@
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+struct main0_out
+{
+    float4 gl_Position [[position]];
+};
+
+vertex main0_out main0()
+{
+    main0_out out = {};
+    out.gl_Position = float4(1.0);
```

Comment:
Could you add a write to gl_PointSize here so we can test that writing multiple builtins works?

Reply:
Done.